app/Models/BankAccount.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class BankAccount extends Model
+{
+    use SoftDeletes;
+
+    protected $table = "bankaccount";
+    
+    protected $primaryKey = "bank_id";
+
+    protected $fillable = [ 
+        'user_id', 
+        'bank_name',
+        'status',
+    ];
+
+    public function payments()
+    {
+        return $this->hasMany('App\Models\Payment', 'bank_id', 'bank_id');
```

Comment:
Payment::class
sao chỗ này anh comment miết vẫn k sửa vậy em? 
fix hết toàn bộ đi